netbox/utilities/context_processors.py

```diff
@@ -8,3 +11,23 @@ def settings(request):
     return {
         'settings': django_settings,
     }
+
+
+def latest_version(request):
```

Comment:
This will need a unit test. We'll need to ensure the test uses dummy data and does not actually reach out to GitHub for every run.